src/plugins/imagery/lib/eventHelpers.js

```diff
@@ -0,0 +1,98 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2021, United States Government
```

Comment:
there are already similar files in `openmct/src/plugins/plot/lib/eventHelpers.js` and `openmct/src/plugins/summaryWidget/src/eventHelpers.js` wondering if we can reuse them.